scap/content/guides/openscap/build-content.log

```diff
@@ -1,5 +1,4 @@
 Cloning into '/tmp/ComplianceAsCode/content'...
-Updating files:  85% (9343/10987)
Updating files:  86% (9449/10987)
Updating files:  87% (9559/10987)
Updating files:  88% (9669/10987)
Updating files:  89% (9779/10987)
Updating files:  90% (9889/10987)
Updating files:  91% (9999/10987)
Updating files:  92% (10109/10987)
Updating files:  93% (10218/10987)
Updating files:  94% (10328/10987)
Updating files:  95% (10438/10987)
Updating files:  96% (10548/10987)
Updating files:  97% (10658/10987)
Updating files:  98% (10768/10987)
Updating files:  99% (10878/10987)
Updating files: 100% (10987/10987)
Updating files: 100% (10987/10987), done.
 /tmp/ComplianceAsCode/content /
 
 Tag to build: v0.1.70
```

Comment:
Looks like it's the same version.  Can you re-run the content update?  I just merged the PR that it's the version to 0.1.71.

Reply:
Updated branch and reran with new version.